src/components/CalltoActionButton.js

```diff
@@ -1 +1,43 @@
-//Gets passed a prop LinkTo that leads to the desired section, exists in two different styles based on its props (https://emotion.sh/docs/styled)
\ No newline at end of file
+import styled from "@emotion/styled"
+import React from "react"
+
+const ButtonLarge = styled.button`
+  border: solid 3px transparent;
+  border-radius: 100rem;
+  background-image: linear-gradient(
+      rgba(255, 255, 255, 0),
+      rgba(255, 255, 255, 0)
+    ),
+    linear-gradient(101deg, #78e4ff, #ff48fa);
+  background-origin: border-box;
+  background-clip: content-box, border-box;
+  padding: 0.5rem 3rem;
+  box-shadow: 2px 1000px 1px #111 inset;
+  color: white;
+  font-family: inherit;
+
+  & :hover {
+    animation: pulse 3s;
+    box-shadow: 0 0 0 2em rgba(#fff, 0);
+  }
+
+  @keyframes pulse {
+    0% {
+      box-shadow: 0 0 0 0 var(--gradient);
+    }
+  }
+`
+
+const ButtonSmall = styled(ButtonLarge)`
```

Comment:
Are you creating two Buttons?

Reply:
I'm creating the same button with different stylings. I fixed the component using dynamic css as Leon suggested 👍